### DISCUSSION TITLE
FontFreeze for baking in stylistic sets

### DISCUSSION THREAD

**Discussion by MuTsunTsai** [tonsky/FiraCode#1448] [Show and tell]
Hello, I would like to bring to maintainers' attention that, besides the opentype-feature-freezer mentioned in the wiki, I've made a new tool for permanently activating font features. It's called FontFreeze and can be used directly from the browser here:
https://mutsuntsai.github.io/fontfreeze/
opentype-feature-freezer is incapable of dealing with ligature features, but FontFreeze can handle those as well!
It would be great if this could be added to the wiki. Thank you!

**Comment by tonsky** [comment]
Done! https://github.com/tonsky/FiraCode/wiki/How-to-enable-stylistic-sets
